python/ribasim/pyproject.toml

```diff
@@ -31,18 +31,9 @@ dynamic = ["version"]
 [project.optional-dependencies]
 tests = ["pytest", "pytest-xdist", "pytest-cov", "ribasim_testmodels"]
 
-[tool.setuptools]
-zip-safe = true
-
-[tool.setuptools.dynamic]
-version = { attr = "ribasim.__version__" }
-
-[tool.setuptools.packages.find]
-include = ["ribasim", "ribasim.*"]
-
-[tool.setuptools.package-data]
-"ribasim" = ["py.typed"]
```

Comment:
I'm not sure if this is required. The whl and sdist folders were the same between setuptools and hatchling

Reply:
So the `py.typed` file is still included?
In that case it should be fine

Reply:
yes it is!